jdbc/src/main/java/software/amazon/timestream/jdbc/TimestreamColumnsResultSet.java

```diff
@@ -134,7 +134,7 @@ protected boolean doNextPage() throws SQLException {
     }
 
     // Get the columns for the next table.
-    curDatabase = tablesResult.getString(1);
+    curDatabase = tablesResult.getString(2);
```

Comment:
Can ordinals be defined as constants and referred in line137-138 instead of relying on magic numbers?